src/js/media-live-button.js

```diff
@@ -44,6 +44,16 @@ slotTemplate.innerHTML = /*html*/`
   <slot name="spacer">&nbsp;</slot><slot name="text">LIVE</slot>
 `;
 
+const updateAriaAttributes = (el) => {
+  const isPaused = el.mediaPaused || !el.mediaTimeIsLive
```

Comment:
`isPaused` isn't the best name to represent `!el.mediaTimeIsLive` since the video is not paused in that case. `isNotLive` might be clearer. `isPausedOrNotLive` would be most clear and accurate probably.

Reply:
I've updated the variable name :)